platform/security/security-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -23,5 +23,5 @@
     </bean>
 
```

Comment:
It doesn't look like you are injecting your new constructor args in your blueprint.